### ISSUE TITLE
Different image gamma between 2.77 and 2.78.

### ISSUE THREAD

**Issue by michax** [bdancer/blender-for-vray#216]
Render output image in 2.78 using same configuration is much darker and has lower gamma. 
In both cases I've sRGB enabled. 

left: 2.77
right: 2.78
![Gamma difference](http://animuj.com/lab/vray/gamma-difference.JPG)

**Comment by poseidon4o** [comment]
Since couple of nightlies the 'sRGB' option has been removed from Render -> System -> Exporter. Now the exporter uses the blender setting: Scene -> Color Management -> Display Device.

Does setting the correct value in Color Management produce the correct gamma?

**Comment by michax** [comment]
Ok, I figured it out. For some reason 2.78 defaulted to "Color Mapping" mode without Gamma.
So whenever I opened 2.77, Gamma value (which is set to 2.2 as default) was applied. In 2.78 it wasn't. 

![Color Mapping](http://animuj.com/lab/vray/color-mapping.JPG)

Thanks for info about sRGB. I didn't know it was moved to Color Management. Make sense. 
For my previous tests I just enabled it manually. 

As for the issue itself. I cannot reproduce it after saving file with Color Mapping & Gamma mode. Both version render picture with same gamma value now. 

Closing.
